website/utils/test/url_utils.test.ts

```diff
@@ -380,6 +382,8 @@ describe("Channel state serialization", () => {
         useControlPoints: false,
         controlPoints: [],
         ramp: [0, 255],
+        lockPlotToDataRange: true,
+        plotMax: 255,
```

Comment:
Should this be serialized to the URL?

Reply:
Maybe worth adding a TODO here at the very least

Reply:
My thought was no, since these are configuration parameters for how a UI element gets drawn, and are never passed down to the vole-core level to modify how the image is actually displayed. Though `TfEditor` may be on its way to being an actual data display in its own right, rather than just a control element. In that case I think it would make more sense to preserve these parameters. I'll give it a TODO and an issue.

Reply:
Created #384 for this